src/main.js

```diff
@@ -26,9 +27,51 @@ router.beforeEach((to, from, next) => {
     }
 });
 
+const store = new Vuex.Store({
```

Comment:
Zastąpmy to zwykłym serwisem, tak jak mamy resztę. Wystarczy nam zwykły ThemeService, która ma metodę odwzorowujące to zachowania. Nawet składnia będzie prostsza:)